gatsby-config.esm.js

```diff
@@ -61,6 +61,33 @@ module.exports = {
         // Exclude pages that just redirect.
         // Exclude our secret test page
         excludes: ['/help', '/contact', '/teamseas/atlantis'],
+        query: `
+        {
+          site {
+            siteMetadata {
+               siteUrl
+            }
+          }
+          allSitePage {
+            nodes {
+              path
+              context {
+                referrer {
+                  id
+                }
+              }
+            }
+          }
+        }`,
+        resolvePages: ({ allSitePage: {nodes: allPages}}) => allPages,
+        filterPages: (page, excludedRoute) => page.context.referrer !== null || page.path.slice(0, -1) === excludedRoute,
```

Comment:
The slice is for removing trailing slashes in the page.path.

Reply:
Please add this as a code comment.